backend/app/Http/Controllers/UserController.php

```diff
@@ -36,4 +38,22 @@ public function login(Request $request)
 
         return LoginResource::make(array_merge($user->toArray(), ['token' => $token]));
     }
+
+    public function updateInfo(UserUpdateInfoRequest $request)
```

Comment:
use resourceful routing methods here

refer to https://laravel.com/docs/8.x/controllers#actions-handled-by-resource-controller

function should be `update`

Reply:
Updated created new controller for the profile